source/_includes/installation/operating_system.md

```diff
@@ -71,46 +76,96 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
+
+  **Method 2**: With this method, you write the Home Assistant Operating disk image directly onto a boot medium. The steps are a bit more complex. If you have non-removable internal mediums (for example because you are using a laptop) or do not have the necessary adapter (for example an USB to eMMC adapter) use method 1 instead.
+
+### Method 1: Installing HAOS via Ubuntu booting from a USB flash drive
+
+#### Required material
+
+- Computer
+- The target x86-64 hardware, on which you want to install the Home Assistant Operating System (HAOS)
+- USB flash drive (USB thumb drive is sufficient, it should be at least 4&nbsp;GB in size)
+- Internet connection
+
+#### To install HAOS via Ubuntu from a USB flash drive
+
+1. **Notice**: This procedure will write the Home Assistant Operating System onto your device.
+   - This means you will lose all the data as well as the previously installed operating system.
+   - Back up your data before continuing with the next step.
+2. Create a *live operating system* on a USB flash drive:
+   - Follow the ([Ubuntu instructions](https://ubuntu.com/tutorials/try-ubuntu-before-you-install) on writing an Ubuntu iso file onto a USB device.
+3. Insert the USB flash drive into the system on which you want to run Home Assistant.
+   - Boot the live operating system.
+4. When prompted, make sure to select **Try Ubuntu**. This runs Ubuntu on the USB flash device.
+   - The system then starts Ubuntu.
+   - Connect your system to your network and make sure it has internet access.
+5. In Ubuntu, open a browser and open [this procedure](/installation/generic-x86-64).
+   - From there, [download the image][generic-x86-64] to your computer.
+6. In Ubuntu, in the bottom left corner, select **Show Applications**.
+7. In the applications, search and open **Disks** and start restoring the HOAS image:
+   1. In **Disks**, on the left side, select the internal disk device you want to install HAOS onto.
+   2. On top of the screen, select the three dots menu and select **Restore Disk Image...**.
+      ![Restore disk image: select three dots menu](/images/installation/ubuntu_restore_disk_image.png)
+   3. Select the image you just downloaded.
+      ![Restore disk image: select image](/images/installation/select_haos.png)
+   4. Select **Start Restoring...**.
+      ![Restore disk image: start restoring](/images/installation/start_restoring.png)
+   5. Confirm by selecting **Restore**.
+      ![Restore disk image: select Restore](/images/installation/restore.png)
+      - If you are getting an **Error unmounting filesystem** error message, stating that the **target is busy**:
+      - Most likely, you are running Ubuntu on your internal disk. Instead, you need to run it on your stick.
+        - Go back to step 3 and during start up, make sure you select **Try Ubuntu** (and NOT **Install Ubuntu**).
+   6. In the partitions overview, you should now see the **hassos-boot** partition.
```

Comment:
Afaik, the `hassos-boot` partition only really becomes visible once flashing succeeded. We can be more generic here:
```suggestion
   6. In the partitions overview, you should now see restore operating in progress.
```